lisa/runners/lisa_runner.py

```diff
@@ -354,8 +354,16 @@ def _run_test_task(
                 f"because after test case '{test_result.name}', "
                 f"node(s) cannot be accessible."
             )
-            # check panic when node(s) in bad status
-            environment.nodes.check_kernel_panics()
+            try:
+                # check panic when node(s) in bad status
+                environment.nodes.check_kernel_panics()
```

Comment:
if there is kernel_panic, and the test result is passed, it should be marked as failed.